apps/store/themes/store/css/theme-categorization.min.css

```diff
@@ -0,0 +1,136 @@
+.radio .helper:before,.radio>input[type=radio]:checked+.helper:after,.switch .helper:before,.switch>input[type=checkbox]+.helper:after,.tree-view.tree-view-lines ul li:before,.tree-view.tree-view-lines ul:before {
```

Comment:
Why have we named this file as min when it is not minified?  (Is this an external library?)